modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -4225,16 +4225,6 @@ else if (StringUtil.equals(
 
 				return;
 			}
-
-			Map<String, Serializable> originalValues = objectEntry.getValues();
-
-			if (!Objects.equals(
-					GetterUtil.getLong(originalValues.get(entry.getKey())),
-					GetterUtil.getLong(entry.getValue()))) {
-
-				throw new ObjectEntryValuesException.
-					UnmodifiableAccountEntryObjectField(objectField.getName());
-			}
```

Comment:
hey @joseabelenda could you please remove this test [here](https://github.com/liferay/liferay-portal/blob/d2b300c6bb680325910b3f612b5cd30576c82384/modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java#L2397-L2404)?

Thank you!